Procfile

```diff
@@ -1,3 +1,3 @@
-web: bundle exec rails server
+web: bundle exec rails server -b 0.0.0.0
```

Comment:
yep, hence the WIP. I just wanted to quickly commit this and let others comment. I just need to do that locally while I tested on android/ios devices. I'll add a proper commit message and cleanup any fixes ASAP.